src/panels/config/automation/ha-config-automation.ts

```diff
@@ -47,6 +47,11 @@ class HaConfigAutomation extends HassRouterPage {
       },
       edit: {
         tag: "ha-automation-editor",
+        load: () => import("./ha-automation-editor"),
```

Comment:
Remove the import on line 12 if we are going to lazy load it.
<https://github.com/home-assistant/frontend/pull/8608/files#diff-dce6345c2be843348904090388b5c5fbfccbeb1cd457bf8e7cf6d510523a8739R12>